cpp/src/strings/convert/convert_floats.cu

```diff
@@ -132,8 +140,6 @@ struct string_to_float_fn {
   __device__ FloatType operator()(size_type idx)
   {
     if (strings_column.is_null(idx)) return static_cast<FloatType>(0);
-    // the cast to FloatType will create predictable results
-    // for floats that are larger than the FloatType can hold
     return static_cast<FloatType>(stod(strings_column.element<string_view>(idx)));
```

Comment:
Recommend re-instating this comment or a rephrasing of it perhaps since this was a vital part of the code behavior.